currency/index.ts

```diff
@@ -15,20 +15,24 @@ const getStxFiatEquivalent = (stxAmount: BigNumber, stxBtcRate: BigNumber, btcFi
 const getBtcFiatEquivalent = (btcAmount: BigNumber, btcFiatRate: BigNumber): BigNumber =>
   satsToBtc(btcAmount).multipliedBy(btcFiatRate);
 
+const getFiatBtcEquivalent = (fiatAmount: BigNumber, btcFiatRate: BigNumber): BigNumber =>
+  new BigNumber(fiatAmount.dividedBy(btcFiatRate).toFixed(8));
```

Comment:
fiatAmount is already a BigNumber, and do we need toFixed(8) here, if we're also returning a BigNumber?

```suggestion
  fiatAmount.dividedBy(btcFiatRate);
```

Reply:
also should we deprecate the function getBtcEquivalent below? or did you specifically want a new function that did toFixed(8) as well?

Reply:
Yeah, we do. 

e.g. a btc is worth 3USD and I want to send 1USD
1/3=0.333333333333333...
If we had to test it, we'd get:
```
> a=new BigNumber('1')
BigNumber { s: 1, e: 0, c: [ 1 ] }
> b=a.div('3')
BigNumber { s: 1, e: -1, c: [ 33333333333333, 33333300000000 ] }
> b.toString()
'0.33333333333333333333'
```

0.33333333333333333333 is not a valid btc value because a bitcoin can only be divided into 100 million sats, so we need to truncate it to 8 digits after the decimal.

Reply:
Ooops, I didn't see the other function 😅 I think the name of the new one makes more sense for consistency with the other function names. wdyt?

Reply:
yeah agree on the naming.

> 0.33333333333333333333 is not a valid btc value because a bitcoin can only be divided into 100 million sats, so we need to truncate it to 8 digits after the decimal.

I see, OK